src/main/java/mate/academy/spring/model/Role.java

```diff
@@ -0,0 +1,46 @@
+package mate.academy.spring.model;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import org.springframework.security.core.GrantedAuthority;
+
+@Entity
+@Table(name = "roles")
+public class Role {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Enumerated(value = EnumType.STRING)
+    private RoleName roleName;
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public RoleName getRoleName() {
+        return roleName;
+    }
+
+    public void setRoleName(RoleName roleName) {
+        this.roleName = roleName;
+    }
+
+    public enum RoleName implements GrantedAuthority {
+        ADMIN,
+        USER;
+
+        @Override
+        public String getAuthority() {
+            return "ROLE_" + this.name();
+        }
+    }
```

Comment:
Let's try to implement this in more simple way:
```suggestion
    public enum RoleName {
        ADMIN, USER;
    }
```